HonorAmongThieves/ReactApp/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "HonorAmongThieves.ReactApp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.1.3",
+    "jquery": "^3.4.1",
+    "merge": "^1.2.1",
+    "oidc-client": "^1.9.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-router-bootstrap": "^0.25.0",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.4.1",
+    "reactstrap": "^8.4.1",
+    "rimraf": "^2.6.2"
+  },
+  "devDependencies": {
+    "ajv": "^6.9.1",
+    "cross-env": "^5.2.0",
+    "typescript": "^3.7.5",
```

Comment:
this is a fairly old version of typescript, the latest is `4.4.4`. I'd recommend upgrading to 4.x, especially for [this](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#unknown-on-catch)